src/main/java/com/code_intelligence/jazzer/mutation/combinator/MutatorCombinators.java

```diff
@@ -149,6 +170,14 @@ public void mutateInPlace(T reference, PseudoRandom prng) {
         mutators[prng.indexIn(mutators)].mutateInPlace(reference, prng);
       }
 
+      @Override
+      public void crossOverInPlace(T reference, T otherReference, PseudoRandom prng) {
+        // TODO: Verify if all, some or one mutator should be used.
```

Comment:
Given that this will usually be used to combine other combinators, e.g. `mutateProperty`, it seems reasonable to not make any random choices here and cross over every mutator. While I could see reasons for only calling cross over on a random subset, I think that always calling cross over on a single mutator wouldn't make much sense.

Reply:
I agree, removed the todo for now.